# 09 Anomaly flags

Do RWTC days that sit more than four robust z-scores from the pre-2020 center lead larger five-print absolute moves?

Holdout starts 2020-01-01. Verdict: held.

This study reads stored series. It does not add a derived column.

In [1]:
import json
from pathlib import Path
studies = json.loads(Path('data/studies.json').read_text())
study = next(item for item in studies['studies'] if item['slug'] == 'anomalies')
print(study['verdict'])
print(study['verdict_line'])
print(study['table']['caption'])
for row in study['table']['rows']:
    print(' | '.join(row))


held
Train median daily log return is 0.071%. MAD scale is 1.852%. Holdout days with |z| above 4: 53. Their mean absolute five-print return is 16.789%, against 4.483% on the other holdout days. The flagged days were followed by larger absolute moves. The holdout contains the 2020 break, and a large print often sits next to another large print.
Holdout RWTC days scored with the frozen train center and MAD
|z| > 4 | 53 | 16.789%
The rest of the holdout | 1625 | 4.483%


## Method

Holdout days with |z| above 4 are the flags. The bar is a higher mean absolute five-print return on flags than on the other holdout days.

## Result

Train median daily log return is 0.071%. MAD scale is 1.852%. Holdout days with |z| above 4: 53. Their mean absolute five-print return is 16.789%, against 4.483% on the other holdout days. The flagged days were followed by larger absolute moves. The holdout contains the 2020 break, and a large print often sits next to another large print.